src/PIL/EpsImagePlugin.py

```diff
@@ -333,22 +341,38 @@ def _read_comment(s: str) -> bool:
                 else:
                     break
 
-                self._size = columns, rows
-                return
+                # Read the columns and rows after checking the bit depth and mode
```

Comment:
```suggestion
                # Parse the columns and rows after checking the bit depth and mode
```
In a Pillow context, I'm very used to thinking of 'read' as meaning 'read from file', which isn't what's happening here.